themes/digital.gov/src/scss/new/_resources.scss

```diff
@@ -1,12 +1,5 @@
 @use "uswds-core" as *;
 
-// .paper sets the background-color for the page
-.resources.section {
-  .paper {
-    @include u-bg("blue-warm-5");
-  }
-}
```

Comment:
Removed `.resources.section .paper` since it is over-ridden on `resources/list.html` and sets background color on `topics-button-list` section which we do not want.

Reply:
@nick-mon1 @clmedders, unresolving the comment as it's useful information for reviewer.